_content/news/_posts/2018-10-19-kerstkampen.markdown

```diff
@@ -1,18 +1,17 @@
 ---
 layout: post
 title:  "Basketbal voor de jeugd tijdens de kerstperiode"
-cover: ![coverfoto](/BasketLummen/Website/_content/albums/2018-10-13-Basket-Lummen-DSEA-Sint-Katelijne-Waver-DSEA/_thumbnails/IMG_6412.png)
+cover: ![Jeugd](Website/_content/albums/2018-10-13-Hasselt-BT-G12B-Basket-Lummen-G12C/_normalized/IMG_6268.jpg)
```

Comment:
De inhoud tussen de --- is geen markdown, maar YAML, Dit path moet /albums/2018-10-13-Hasselt-BT-G12B-Basket-Lummen-G12C/_normalized/IMG_6268.jpg zijn, en niet de markdown notatie